src/main/java/com/Model/UserPassword.java

```diff
@@ -0,0 +1,28 @@
+package com.Model;
+
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import javax.persistence.Embeddable;
+import java.io.Serializable;
+
+@Embeddable
+public class UserPassword implements Serializable {
+
+    private String encryptedPassword;
+
+    public UserPassword (){
```

Comment:
Not sure this default should be allowed. Since this is not an @Entity class, maybe we don't need to provide a default constructor at all.